spec/controllers/coronavirus_form/offer_staff_type_controller_spec.rb

```diff
@@ -58,14 +58,14 @@
       post :submit, params: params
 
       expected_number_hash = {
-        cleaners_number: params[:cleaners_number],
-        developers_number: params[:developers_number],
-        medical_staff_number: params[:medical_staff_number],
-        office_staff_number: params[:office_staff_number],
-        security_staff_number: params[:security_staff_number],
-        trainers_number: params[:trainers_number],
-        translators_number: params[:translators_number],
-        other_staff_number: params[:other_staff_number],
+        cleaners_number: Integer(params[:cleaners_number]),
+        developers_number: Integer(params[:developers_number]),
+        medical_staff_number: Integer(params[:medical_staff_number]),
+        office_staff_number: Integer(params[:office_staff_number]),
+        security_staff_number: Integer(params[:security_staff_number]),
+        trainers_number: Integer(params[:trainers_number]),
+        translators_number: Integer(params[:translators_number]),
+        other_staff_number: Integer(params[:other_staff_number]),
```

Comment:
I wonder if any of these fields are permitted to be `nil`, which I suppose would raise an unhandled `TypeError` here and a 500 to the user.

Reply:
For handling these values in the form I actually use the the `String#to_i` which defaults to 0 rather than error/nil, so they should never have a 500 here.